README.md

```diff
@@ -81,3 +81,21 @@ For the information on how to run tests and configure them, go to the [`tests`](
 ## Troubleshooting
 
 > **TIP:** To solve most of the problems with Busola development, clear the browser cache or do a hard refresh of the website.
+
+## Busola in Docker against a k3d cluster
+
+Due to the fact the k3d cluster's API server is exposed on `0.0.0.0` address, connectivity problems may occur.
+
+- For Docker Desktop for Mac and Windows, pass `DOCKER_DESKTOP_CLUSTER=true` on dockerized Busola startup.
+
+```bash
+docker run -p 3001:3001 -e DOCKER_DESKTOP_CLUSTER=true busola/local:latest
+```
```

Comment:
```suggestion
   ```
```

Reply:
Wait, but this way all text after this block will be rendered as a bash script...?
![image](https://user-images.githubusercontent.com/10504661/133744787-75c5d050-78ce-4982-8269-e301e09a933b.png)